contributing/writing-tests.md

```diff
@@ -30,6 +30,18 @@ export HCP_CLIENT_ID=...
 export HCP_CLIENT_SECRET=...
 ```
 
+For some tests AWS credentials are also required in order to create "customer"
+resources for testing certain HCP features (network peerings and TGW attachments):
+
+```sh
+export AWS_ACCESS_KEY_ID=...
+export AWS_SECRET_ACCESS_KEY=...
+export AWS_SESSION_TOKEN=...
+```
+
+Note that this AWS account **MUST NOT** be the same AWS account that is being used by
```

Comment:
Thanks for adding this but I think we can leave it out, since this is something only an internal employee can run into and it might confuse external contributors.

Reply:
that was my original reason to leave it out but thinking about it now - in the majority of the cases these tests will be probably run by someone from HCP...

Reply:
added "Note for HCP developers" here